src/couch/src/couch_os_process.erl

```diff
@@ -258,6 +259,12 @@ bump_time_stat(Stat, USec) when is_atom(Stat), is_integer(USec) ->
     couch_stats:increment_counter([couchdb, query_server, calls, Stat]),
     couch_stats:increment_counter([couchdb, query_server, time, Stat], USec).
 
+bump_volume_stat(ddoc_filter = Stat, Docs) when is_atom(Stat), is_list(Docs) ->
+    couch_stats:increment_counter([couchdb, query_server, volume, Stat], length(Docs));
+bump_volume_stat(_, _) ->
+    %% TODO: handle other stats?
+    ok.
+
```

Comment:
At first it seems odd to count filtered docs in the couch_os_process, it seems this would go somewhere at the higher level. We don't count mapped docs and such here for instance?

In general wouldn't filtered docs match the changed feed processed docs for a particular docs, maybe we can skip the stat altogether?